bin/offline-dbg

```diff
@@ -133,6 +142,143 @@ EOF
     echo "**************************************************************"
 }
 
+get_highest_ofp_version() {
+    # If no protocol listed, default to OpenFlow14 (enabled by default on ovs 2.8 on)
+    hex_ofp_version=$(cat ${CMD_DIR}/ovs-ofctl_-t_5_--version | grep "OpenFlow versions" | grep -oP [0-9]*x[0-9]* | tail -1)
+    version=${open_flow_protocols[$hex_ofp_version]}
+    if [ -z "$version" ]; then
+        echo "OpenFlow14"
+    else
+        echo $version
+    fi
+}
+
+do_collect-sos() {
+    OPTIND=1
+    while getopts "h" opt; do
+        case ${opt} in
+            h)
+                usage
+                exit 0
+                ;;
+        esac
+    done
+    shift $(((OPTIND -1)))
```

Comment:
Sorry I did not express my point clearly here.
No need to call getopts only of "-h" since the main call to getopts already handles this case

Reply:
Note that other "collect-X" functions don't call getopts unless they provide options other than "h"